.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+
+If related to user experience, use the format:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+If related to development, please provide relevant configuration details necessary to understand your problem including any relevant traces, logs, or otherwise.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Smartphone (please complete the following information):**
```

Comment:
Authenticator and/or CP app log?

Reply:
Since this is first party and most of our bug reports through this channel will be 3rd party, we'll avoid putting this guidance here.